mailing-list-api/server.js

```diff
@@ -0,0 +1,60 @@
+import express from "express";
+import mailinglist from "./mailing-lists.js";
+const PORT = 3000;
+
+const app = express();
+app.use(express.json());
+app.disable("x-powered-by");
+
+const mailList = mailinglist;
```

Comment:
Another line that doesn't make too much sense to me - is there any particuluar reason that the var needs reassigned?